### ISSUE TITLE
Bug in compiler output for plural keyword getter in some languages.

### ISSUE THREAD

**Issue by zdhr** [tinganho/l10ns#156]
Hi,
plural form for czech, slovak and probably other slavic languages isn't working properly. It uses **few** only when n=2, instead of when n=2..4. While debuging this, I found out, that compiler output for plural keyword getter is most likely wrong.

This is what i get:
```php
else if((2 >= i && i <= 4) && v === 0) {
	return 'few';
}
```
It returns few only when i === 2, when I change it to this, then it works correctly:
```php
else if((i >= 2 && i <= 4) && v === 0) {
	return 'few';
}
```

But great job on this library otherwise, looking forward for a fix, so i can use it.

**Comment by tinganho** [comment]
Hi @zdhr you are right this is a bug. I will try to fix it ASAP.

**Comment by tinganho** [comment]
This should now be fixed in v2.4.11
